tsd/src/tsd/objects/Geometry.cpp

```diff
@@ -21,6 +21,9 @@ const Token isosurface = "isosurface";
 const Token quad = "quad";
 const Token sphere = "sphere";
 const Token triangle = "triangle";
+#ifdef USE_NEURAL_GRAPHICS_PRIMITIVES
```

Comment:
Let's always define things that don't have external dependencies, such as this `Token`. The only thing we will conditionally compile in is the code which depends on `Torch`, otherwise we should keep the rest.